infra/deploy.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import urllib, json, subprocess
+
+url = 'https://registry.hub.docker.com/v2/repositories/wellcome/wellcomecollection/tags/'
+response = urllib.urlopen(url)
+data = json.loads(response.read())
```

Comment:
If you can install Python packages, you may prefer to use `requests`, which has a much nice interface:

```python
import requests

resp = requests.get('https://registry.hub.docker.com/v2/repositories/wellcome/wellcomecollection/tags/')
resp.raise_for_status()  # Check we got a 200, not a 4xx or 5xx
data = resp.json()
```

Bonus: the same code works in Python 2 or 3.

Edit: I can see you're installing awscli in the container, so I’d really recommend requests.

Reply:
This isn't actually on our CI. It's local. I could do but it requires another device step, whereas I'm trying to make things simpler. When Devs start complaining about 500 errors, we can look into better response handling.

Reply:
It does raise the point, I'd like to chat about how we might cache pythons deps in Circle.

Reply:
It does raise the point, I'd like to chat about how we might cache pythons deps in Circle.

Reply:
On a Unix system, pip will cache to `~/.cache/pip`, so preserving that directory between runs would be a good start.